src/MeasureToolbox/support_sums.jl

```diff
@@ -56,15 +56,17 @@ end
 An efficient wrapper for [`support_sum`](@ref) please see its doc string for
 more information.
 """
-macro support_sum(expr, prefs, args...)
-    _error(str...) = InfiniteOpt._macro_error(:support_sum, (expr, prefs, args...), 
-                                              str...)
-    extra, kwargs, _, _ = InfiniteOpt._extract_kwargs(args)
-    if length(extra) > 0
-        _error("Unexpected positional arguments." *
-               "Must be of form @support_sum(expr, prefs, kwargs...).")
-    end
+macro support_sum(args...)
+    error_fn = InfiniteOpt.JuMPC.build_error_fn(:support_sum, args, __source__)
+    pos_args, kwargs = InfiniteOpt.JuMPC.parse_macro_arguments(
+        error_fn, 
+        args, 
+        num_positional_args = 2,
+        valid_kwargs = [:label]
+    )
```

Comment:
Nice. The uniformity of error messages is a win

Reply:
Yep!